src/models/Settings.js

```diff
@@ -314,6 +314,9 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
     },
 
     setAuthClient: function (authClient) {
+      if (authClient && authClient.options && authClient.options.headers) {
+        authClient.options.headers['Accept-Language'] = this.get('languageCode');
```

Comment:
We do not have to add to the existing list of accept-language header value? Is replacing fine?

Reply:
Discussed over slack: 

### TLDR; 
Add another fallback locale appended along with `languageCode` while setting `Accept-Language` header.


### Details:
Typically the browser sends a list of languages with their weights as a part of `Accept-Language` header 

Eg: `Accept-Language: en,es;q=0.9,fr;q=0.8,el;q=0.7,ru;q=0.6,nl;q=0.5,pt-BR;q=0.4,pt;q=0.3,es-US;q=0.2,en-US;q=0.1,pt-PT;q=0.1,zh;q=0.1,zh-CN;q=0.1,zh-TW;q=0.1,zh-HK;q=0.1`)

Which means english is preferred local for response, but es is the next preferred locale on the list.

For this PR, instead of just sending the `languageCode`, we will try to also send a fallback locale along with the `languageCode`. 

This fallback locale will be the first locale from the list of browser languages with a weight of `0.9`.

Ideally we should not have to have to use the fallback locale since the `languageCode` will default to `en` if user tries to set it to a unsupported locale.

Example value 
authClient.options.headers['Accept-Language'] = `es, en;q=0.9`

Where `es` is the languageCode configured in the widget and the preferred locale for the response, but en is the fallback locale.

@santhoshbalakrishnan @mauriciocastillosilva-okta @hor-kanchan-okta let me know you guys think.

Reply:
What's the reason for the fallback? It seems like the point of this PR is to get consistency between the widget language and any text sent back in an API response - if the widget thinks it should display in English (because they didn't override or because they overrode with an unsupported language), why send another option?

There are probably some other edge cases to think about:
- What if they specify their own translations for a language we don't support? This will work for anything that's bundled in the widget, but would be one of the cases where we don't have translations in the backend - is this the reason for the fallback?
  - And if this is the case, are we okay with changing the fallback from what it currently is (english) to the next-best option from the browser?
-  Might not be relevant, but the widget is currently setup in a way that language can be changed on the fly - in this commit it only gets set when initializing the auth client. Might be worth thinking about making this more dynamic in authJs.

Reply:
> This fallback locale will be the first locale from the list of browser languages with a weight of 0.9

For clarity, would change that to `first locale from the list of browser languages *that we support*`

Will wait for @mauriciocastillosilva-okta and @hor-kanchan-okta 's comments.

Reply:
@rchild-okta 

>And if this is the case, are we okay with changing the fallback from what it currently is (english) to the next-best option from the browser?

Ya, that's what we discussed. More like next best option that we have in our supported list of languages.